web/styles/dark_theme.css

```diff
@@ -196,7 +196,7 @@
     #userlist-toggle-button,
     .header,
     #message_view_header {
-        background-color: hsl(0deg 0% 13%);
+        background-color: var(--color-background);
```

Comment:
I think this should be `--color-background-navbar`

Reply:
`--color-background-navbar` for the dark theme which has `hsl(0deg 0% 13%);` was not used anywhere in the codebase.
So changed it to `hsl(0deg 0% 11%);` which is equal to `--color-background` and used var(`--color-background-navbar`) here.

Reply:
No, it was used here before you changed it. You want the dark mode navbar to be `hsl(0deg 0% 13%);` because that's what the designs say to do.

Reply:
Oh, so should I make the other part's background color the same as `hsl(0deg 0% 13%)`.